deploy/crds/operators.coreos.com_servicebindings_crd.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     - sbrs
     singular: servicebinding
   scope: Namespaced
+  preserveUnknownFields: false
```

Comment:
we need to specify the annotation on `servicebinding_types.go`, otherwise, this is going to removed by the next invocation of `operator-sdk generate crds`

Reply:
Ohh true I'll need to add it there as well, nice catch thank you.